jsk_pcl_ros/src/pointcloud_screenpoint_nodelet.cpp

```diff
@@ -132,10 +132,14 @@ void jsk_pcl_ros::PointcloudScreenpoint::onInit()
 
 bool jsk_pcl_ros::PointcloudScreenpoint::checkpoint (pcl::PointCloud< pcl::PointXYZ > &in_pts, int x, int y,
                                                      float &resx, float &resy, float &resz)  {
-  if ((x < 0) || (y < 0) || (x >= in_pts.width) || (y >= in_pts.height)) return false;
+  if ((x < 0) || (y < 0) || (x >= in_pts.width) || (y >= in_pts.height)) {
+    JSK_ROS_WARN_STREAM("Requested point is out of image size.  point: ("
+                        << x << "," << y << ")" << "  size: (" << in_pts.width << "," << in_pts.height << ")");
```

Comment:
Personally I usually use printf, and I use stream in the case:
```
ROS_INFO_STREAM("Number of clusters: " << cluster_indices_msg.clusters.size());
```
or when displaying rosmsg
<img width="1280" alt="screen shot 2016-10-21 at 11 43 54 pm" src="https://cloud.githubusercontent.com/assets/4310419/19602462/4cd75b72-97e8-11e6-9d9a-4df5e3b23045.png">